CHANGELOG.md

```diff
@@ -3,7 +3,125 @@
 All notable changes to this project will be documented in this file. This
 project adheres to [Semantic Versioning](http://semver.org/).
 
-## Unreleased
+## 0.37.0
+
+This release contains a number of fixes and enhancements.
+Please be aware that it also includes a security fix relevant for anyone
+using _optimized bundles_.
+
+This is the first release that includes a binary and a docker image for
+`linux/arm64`: `opa_linux_arm64_static` and `openpolicyagent/opa:0.37.0-static`.
+Thanks to @ngraef for contributing the build changes necessary.
+
+### Strict Mode
+
+There have been numerous checks in the compiler that fall into this category:
```

Comment:
You could read this as if these checks have already been in effect in previous releases.
Consider `There have been numerous possible checks ..`, or similar.
No biggie, as any wrong assumptions are corrected shortly after.